vcs.js

```diff
@@ -47,12 +47,14 @@ module.exports = {
     /**
      * generate current git url
      * @param  {string} root_path - The root directory to perform the scan (default to current)
+     * @param  {string} is_linux  - If the system is on linux or windows
      * @return {string|null}      - return project name as a string if found, null otherwise
      */
-    getProject: function(root_path) {
+    getProject: function(root_path, is_linux) {
         try{
-            var project_name = Common.systemSync('basename \`git rev-parse --show-toplevel\`', root_path);
-            return project_name;
+            var project_name = Common.systemSync('git rev-parse --show-toplevel', root_path);
+            if(is_linux){ return project_name.split('/').slice(-1)[0].replace('\n', ''); }
```

Comment:
Could use path.sep to get the platform specific version of slash.  Then you don't need to pass `is_linux` around
`return project_name.split(path.sep).slice(-1)[0].replace('\n', '');`

https://nodejs.org/docs/latest/api/path.html#path_path_sep

Reply:
neat, i will change